tests/php/Control/HTTPTest.php

```diff
@@ -112,6 +113,79 @@ public function testConfigVary()
         $this->assertEmpty($v);
     }
 
+    public function testDeprecatedVaryHandling()
+    {
+        /** @var Config */
+        Config::inst()->update(
```

Comment:
```diff
- Config::inst()->update(
+ Config::modify()->set(
```

Reply:
Oh is that what the cool kids are using these day? :D Updated.